workspace/team.go

```diff
@@ -0,0 +1,106 @@
+package workspace
```

Comment:
add tests for this file

Reply:
Skipping tests for this file as they are already covered in workspace_teams_test.go